libnmstate/nm/bridge.py

```diff
@@ -140,26 +177,63 @@ def get_info(context, nmdev):
     info[LB.CONFIG_SUBTREE] = {
         LB.PORT_SUBTREE: _get_bridge_ports_info(
             port_profiles_by_name,
             port_names_sysfs,
             vlan_filtering_enabled=bridge_setting.get_vlan_filtering(),
         ),
         LB.OPTIONS_SUBTREE: {
-            LB.Options.MAC_AGEING_TIME: props.ageing_time,
-            LB.Options.GROUP_FORWARD_MASK: props.group_forward_mask,
-            LB.Options.MULTICAST_SNOOPING: props.multicast_snooping,
+            LB.Options.MAC_AGEING_TIME: props["ageing_time"],
+            LB.Options.GROUP_FORWARD_MASK: props["group_fwd_mask"],
+            LB.Options.MULTICAST_SNOOPING: props["multicast_snooping"] > 0,
             LB.STP_SUBTREE: {
-                LB.STP.ENABLED: bridge_setting.props.stp,
-                LB.STP.PRIORITY: bridge_setting.props.priority,
-                LB.STP.FORWARD_DELAY: bridge_setting.props.forward_delay,
-                LB.STP.HELLO_TIME: bridge_setting.props.hello_time,
-                LB.STP.MAX_AGE: bridge_setting.props.max_age,
+                LB.STP.ENABLED: props["stp_state"] > 0,
+                LB.STP.PRIORITY: props["priority"],
+                LB.STP.FORWARD_DELAY: props["forward_delay"],
+                LB.STP.HELLO_TIME: props["hello_time"],
+                LB.STP.MAX_AGE: props["max_age"],
             },
         },
     }
+
+    if nm_version_bigger_or_equal_to("1.25.2"):
+        info[LB.CONFIG_SUBTREE][LB.OPTIONS_SUBTREE].update(
+            {
+                LB.Options.HELLO_TIMER: props["hello_timer"],
+                LB.Options.GC_TIMER: props["gc_timer"],
+                LB.Options.MULTICAST_ROUTER: props["multicast_router"],
+                LB.Options.GROUP_ADDR: props["group_addr"].upper(),
+                LB.Options.HASH_MAX: props["hash_max"],
+                LB.Options.MULTICAST_LAST_MEMBER_COUNT: props[
+                    "multicast_last_member_count"
+                ],
+                LB.Options.MULTICAST_LAST_MEMBER_INTERVAL: props[
+                    "multicast_last_member_interval"
+                ],
+                LB.Options.MULTICAST_QUERIER: props["multicast_querier"] > 0,
+                LB.Options.MULTICAST_QUERIER_INTERVAL: props[
+                    "multicast_querier_interval"
+                ],
+                LB.Options.MULTICAST_QUERY_USE_IFADDR: props[
+                    "multicast_query_use_ifaddr"
+                ]
+                > 0,
+                LB.Options.MULTICAST_QUERY_INTERVAL: props[
+                    "multicast_query_interval"
+                ],
+                LB.Options.MULTICAST_QUERY_RESPONSE_INTERVAL: props[
+                    "multicast_query_response_interval"
+                ],
+                LB.Options.MULTICAST_STARTUP_QUERY_COUNT: props[
+                    "multicast_startup_query_count"
+                ],
+                LB.Options.MULTICAST_STARTUP_QUERY_INTERVAL: props[
+                    "multicast_startup_query_interval"
+                ],
```

Comment:
I have been thinking about this, I think you can create a constant dict with `{nmstate_key:props_key}` this way you can populate the info dict with a for loop. What do you think?

Reply:
Changed to dict.